src/install/lockfile.zig

```diff
@@ -110,6 +106,7 @@ allocator: std.mem.Allocator,
 scratch: Scratch = Scratch{},
 
 scripts: Scripts = .{},
+workspace_paths: std.HashMapUnmanaged(u64, String, IdentityContext(u64), 80) = .{},
```

Comment:
We could go with an ArrayHashMap here since most projects will have < 10 or so workspaces, but it's fine either way

Reply:
Done in https://github.com/oven-sh/bun/pull/1764/commits/3539db0ac230adbc27dcc3fa2d8a28d5c1250989